tomcat9/image.yaml

```
@@ -29,13 +29,13 @@ envs:
     - name: "JBOSS_PRODUCT"
       value: "webserver"
     - name: "JBOSS_WEBSERVER_VERSION"
-      value: "5.2.0"
+      value: "5.3.0"
     - name: "PRODUCT_VERSION"
-      value: "5.2.0"
+      value: "5.3.0"
     - name: "TOMCAT_VERSION"
       value: "9.0.21"
```

```suggestion
      value: "9.0.30"
```

In JWS 5.3 we updated to tomcat version 9.0.30, so we should change this value.